Makefile

```diff
@@ -150,12 +148,16 @@ run-restaurant-backend:
 ### GITHUB RELEASES
 .PHONY: prepare-release
 prepare-release:
-	tar -C target/ -czvf slight-unix.tar.gz release/slight
+	tar -C target/ -czvf slight-ubuntu.tar.gz release/slight
```

Comment:
shouldn't this be `slight-linux-arm64.tar.gz`?

I would recommend put semantic version in the name.

Reply:
done ✅